packages/sdks/src/functions/get-content/generate-content-url.ts

```diff
@@ -45,6 +55,35 @@ export const generateContentUrl = (options: GetContentOptions): URL => {
     }${enrich ? `&enrich=${enrich}` : ''}`
   );
 
+  url.searchParams.set('omit', omit || 'meta.componentsUsed');
+
+  if (fields) {
+    url.searchParams.set('fields', fields);
+  }
+
+  if (typeof includeUnpublished === 'boolean') {
```

Comment:
won't these get possibly overridden below when we loop over all the keys?  I'm worried about all these special handling interfering with the normal flow

Reply:
Instead, should these be done after line  94?

Reply:
And from a quick skim of the old SDK, I see the same thing: looping over arbitrary k/v pairs from `options` is done towards the end of the URL generation logic.